### PR TITLE
Car price pred

### PR DESCRIPTION
**Pull request by nafis-ishrak**

### CODE REVIEW

File: CarPricePrediction/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- `scikit-learn==1.3.0`
+- `pickleshare==0.7.5`
+- `numpy==1.25.1`
+- `streamlit==1.25.0`
+- `pandas==2.0.3`
+
+
+### Files Required
+
+The python script requires a pickle file to be saved in the same directory: `model.pkl` and a `requirements.txt` file that includes the name and versions of all the packages used. The pickle files contains the Random Forest Regressor model, which is loaded into the script using the pickle library.
```
Comment (98sean98) [inline CarPricePrediction/README.md]: - not `model.pkl` but `RandomForestRegressor.pkl`
- minor grammar mistake in "The pickle file contains the Random Forest ..."

File: CarPricePrediction/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+The python script requires a pickle file to be saved in the same directory: `model.pkl` and a `requirements.txt` file that includes the name and versions of all the packages used. The pickle files contains the Random Forest Regressor model, which is loaded into the script using the pickle library. 
+
+<!-- ## Usage
+
+To run the app, simply run the `app.py` script using the following command:
+
+```shell
+ streamlit run app.py --server.address="0.0.0.0"
+```
+ -->
```
Comment (98sean98) [inline CarPricePrediction/README.md]: Is this section intended to be shown?

**Review by 98sean98 — CHANGES_REQUESTED** [review]
Require minor changes

**Comment by 98sean98** [comment]
What's the currency of the `Present Price`? It's important to make this clear. And also the output is still using `lakhs`?

I think everything should either use `lakhs` or not use `lakhs` at all.

Screenshot 2023-07-27 at 11 24 57 AM

**Comment by nafis-ishrak** [comment]
> 

docker image works perfectly. just didnt deploy the updated one on deploifai yet

**Review by 98sean98 — APPROVED** [review]

**Pull request merged**